components/function-runtimes/python39/Dockerfile

```diff
@@ -0,0 +1,161 @@
+#
+# NOTE: THIS DOCKERFILE IS GENERATED VIA "update.sh"
+#
+# PLEASE DO NOT EDIT IT DIRECTLY.
+#
+
+# We use this code to manually use newest alpine and nodejs.
+# Use official build when it will be available.
+
+FROM alpine:3.13.5
+
+# ensure local python is preferred over distribution python
+ENV PATH /usr/local/bin:$PATH
+
+# http://bugs.python.org/issue19846
+# > At the moment, setting "LANG=C" on a Linux system *fundamentally breaks Python 3*, and that's not OK.
+ENV LANG C.UTF-8
+
+# runtime dependencies
+RUN set -eux; \
+	apk add --no-cache \
+# install ca-certificates so that HTTPS works consistently
+		ca-certificates \
+	;
+# other runtime dependencies for Python are installed later
+
+ENV GPG_KEY E3FF2839C048B25C084DEBE9B26995E310250568
+ENV PYTHON_VERSION 3.9.5
+
+
+# expat-dev for alpine:3.13.5 is outdated and vulnerable. We will install it from edge .
+# We can't upgrade fully to 3.14 because it has a dependancy on a specific recent docker version https://gitlab.alpinelinux.org/alpine/aports/-/issues/12396
+
+RUN set -ex \
+	&& apk add --no-cache --virtual .fetch-deps --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main \
+		gnupg \
+		tar \
+		xz \
+		expat-dev \
```

Comment:
`expat-dev` is installed from `edge` to avoid security issues in the version the comes from `3.13.5`.

Reply:
I feel like we're piling up similar replaces that are forced by security. I was wondering how we could keep track of them and review them periodically to check if we can go back to the stable versions. Maybe we could start with a card on the Github board.